File: src/main/java/ar/edu/um/tesoreria/rest/model/ArancelTipo.java
/**
 * 
 */
package ar.edu.um.tesoreria.rest.model;

import java.io.Serializable;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;

/**
 * @author daniel
 *
 */
@Data
@Entity
@Table
@EqualsAndHashCode(callSuper = false)
@NoArgsConstructor
@AllArgsConstructor
public class ArancelTipo extends Auditable implements Serializable {
	/**
	 * 
	 */
	private static final long serialVersionUID = -7609169359202364699L;

	@Id
	@Column(name = "art_id")
	private Integer arancelTipoId;

	@Column(name = "art_descripcion")
	private String descripcion;
	
	private Byte medioArancel;
	private Integer arancelTipoIdCompleto;

}

File: src/main/java/um/tesoreria/core/kotlin/model/ArancelTipo.kt
package um.tesoreria.core.kotlin.model

import jakarta.persistence.Entity
import jakarta.persistence.Table
import jakarta.persistence.Id
import jakarta.persistence.Column
import um.tesoreria.core.util.Jsonifier

@Entity
@Table
data class ArancelTipo(

	@Id
	@Column(name = "art_id")
	var arancelTipoId: Int? = null,

	@Column(name = "art_descripcion")
	var descripcion: String = "",

	var medioArancel: Byte = 0,
	var arancelTipoIdCompleto: Int? = null,
	var habilitado: Byte = 0

) : Auditable() {

	fun jsonify(): String {
		return Jsonifier.builder(this).build()
	}

}